crates/turborepo-telemetry/src/config.rs

```diff
@@ -67,7 +67,7 @@ impl TelemetryConfig {
     }
 
     pub fn new(config_path: AbsoluteSystemPathBuf) -> Result<TelemetryConfig, ConfigError> {
-        debug!("Telemetry config path: {}", config_path);
+        trace!("Telemetry config path: {}", config_path);
```

Comment:
This is being called a billion times becuase `.with_default_config_path()` is called a lot. Generally think when something is called a lot, it should be trace logging. Aggregations are more useful for debug logging